src/coreclr/jit/emitarm64.cpp

```diff
@@ -16804,12 +16924,30 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             break;
 
         case IF_SVE_EA_1A: // ........xx...... ...iiiiiiiiddddd -- SVE broadcast floating-point immediate (unpredicated)
+        case IF_SVE_ED_1A: // ........xx...... ...iiiiiiiiddddd -- SVE integer min/max immediate (unpredicated)
+        case IF_SVE_EE_1A: // ........xx...... ...iiiiiiiiddddd -- SVE integer multiply immediate (unpredicated)
+        {
+            code = emitInsCodeSve(ins, fmt);
+            code |= insEncodeReg_V_4_to_0(id->idReg1());     // ddddd
+            code_t imm8 = (code_t)(emitGetInsSC(id) & 0xFF); // iiiiiiii
```

Comment:
Given everything else is in functions, then this should be moved into a new helper function:
```
code_t imm8 = (code_t)(emitGetInsSC(id) & 0xFF);
code |= (imm8 << 5);
```

Reply:
Got it, fixed.